code-of-conduct.md

```diff
@@ -68,9 +68,9 @@ members of the project's leadership.
 ## Attribution
 
 This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
-available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
+available at <https://www.contributor-covenant.org/version/1/4/code-of-conduct.html>
 
 [homepage]: https://www.contributor-covenant.org
 
 For answers to common questions about this code of conduct, see
-https://www.contributor-covenant.org/faq
+<https://www.contributor-covenant.org/faq>
```

Comment:
Not sure why `remark` likes adding angle brackets to links... 🤔 could ask and find out?

Reply:
I'd be curious to know what the effect / reasoning of this is (haven't read the commonmark spec lately, so not sure if this even changes anything in the rendering?)

Reply:
I think it depends on the consumer. This falls under [`autolinks` in the commonmark spec](https://spec.commonmark.org/0.29/#autolinks)

GitHub has an [Autolinks extension](https://github.github.com/gfm/#autolinks-extension-) that handles urls without angle brackets

Looks like [remark inserts them](https://github.com/remarkjs/remark/blob/acada43abe624f306545800da0de633063e3cccc/packages/remark-stringify/lib/util/enclose-uri.js) in all instances when they are valid.

They will be inserted automatically as part of the formatted output. There won't be a lint rule or a need to insert them manually